Alert Builder updates for GA and Alert Wizard fixes (#3506)
getsentry/sentry-docs: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx

## 1. Choose Errors or Performance

Select what you want to be alerted about from a number of pre-set alerts under "Errors" or "Performance". Your selection determines some of the alert properties, including whether you're alerted about [errors](/platforms/javascript/usage/#capturing-errors) or [transactions](/product/performance/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate). You can also choose to create a custom metric.

When you've made your selection, click "Set Conditions".

## 2: Filter Events

> Specify which [environment(s)](/platforms/javascript/configuration/environments/) will use this particular alert rule. `Environment` is a Sentry supported tag that you can (and should) add to your SDK. Generally, the [tag](/platforms/javascript/enriching-events/tags/) accepts any value but is targeted to refer to your code deployments' naming convention such as _Development_, _Testing_, _Staging_, or _Production_. The environment drop-down list is populated with the environment tag values available in your [Event Stream](/product/accounts/quotas/manage-event-stream-guide/).

Reviewer: ```suggestion
Specify which [environment(s)](/platforms/javascript/configuration/environments/) will use this alert rule. `Environment` is a Sentry supported tag that you can (and should) add to your SDK, unless your SDK automatically detects the environment. Generally, the [tag](/platforms/javascript/enriching-events/tags/) accepts any value but is targeted to refer to your code deployments' naming convention such as _Development_, _Testing_, _Staging_, or _Production_. The environment drop-down list is populated with the environment tag values available in your [Event Stream](/product/accounts/quotas/manage-event-stream-guide/).
```

Each environment typically necessitates different levels of urgency and workflow. The urgency and workflows you apply to Production alerts might differ from those you apply to alerts originating from your QA environment, for example.

> Add filters to narrow down what you'll be alerted about, such as transactions by URL, tags, or other event properties. Depending on the type of alert you chose, you can also set which tyes of events you want to be alerted about.

Reviewer: ```suggestion
Add filters to narrow down what you'll be alerted about, such as transactions by URL, tags, or other event properties. Depending on the type of alert, you can also set which types of events you want to be alerted about.
```

Manage your alert rule by giving it a descriptive name. A descriptive alert rule name specifies the team affected and the topic of the alert. For example, "Frontend Latency," "Backend Failure Rate," or "Billing Apdex."

> You can choose a team to associate with the alert. When you do so, members of the selected team can edit this alert. Only teams that you're a member of are available for you to choose.

Reviewer: see previous edit.

> You can choose a team to associate with the alert. When you do so, members of the selected team can edit this alert. Only teams that you're a member of are available for you to choose.

Author: ```suggestion
You can choose a team to associate with an alert so that members of the that team can edit this alert. Note that you can only make this association if you are a member of the team.
```
